### DISCUSSION TITLE
[Support] 寻找 Swagger 的 自定义插件 的代码样例

### DISCUSSION THREAD

**Discussion by googoo625** [midwayjs/midway#3952] [Q&A]
### Describe the problem（描述问题）

目的：
在官方组件产生Swagger文档之后，输出之前，再加一些处理。

曾做的努力：
1）config.default.ts里配置Swagger的routerFilter，对一些url做修改，比如去掉全局前缀。没用。
  查官方代码发现url无法修改。这里第二个参数中可以设置ignoreGlobalPrefix=true，但代码里完全没用到，怎么才能起作用呢？

2）搜索能发现自定义插件的一些似是而非的例子，照着做没起作用。

期待得到帮助，谢谢！

### Midway Versions（Midway 版本）

"@midwayjs/swagger": "^3.16.2"

**Comment by czy88840616** [comment]
存在全局前缀也是因为你代码里路由加了全局前缀吧，如果路由改了，那swagger 怎么对应到原始的路由上。

**Comment by googoo625** [comment]
问题提得不太合理，关了。
